jetbrains-core/src/software/aws/toolkits/jetbrains/services/telemetry/TelemetryService.kt

```diff
@@ -133,7 +134,9 @@ class DefaultTelemetryService(settings: AwsSettings) :
     }
 
     override fun sendFeedback(sentiment: Sentiment, comment: String) {
-        publisher.sendFeedback(sentiment, comment)
+        runBlocking {
```

Comment:
Should `sendFeedback` be a blocking call, or should it get queued up with the rest of our metrics and flushed asynchronously?

Reply:
ties into behavior discussion below